modules/op-running-a-pipeline.adoc

```diff
@@ -41,6 +42,7 @@ $ tkn pipeline start build-and-deploy \
     -p deployment-name=pipelines-vote-ui \
     -p git-url=https://github.com/openshift/pipelines-vote-ui.git \
     -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-ui
```

Comment:
`-p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \`, `\` is missing